src/fsharp/FSharp.Core/list.fs

```diff
@@ -223,7 +223,7 @@ namespace Microsoft.FSharp.Collections
 
         [<CompiledName("Pairwise")>]
         let pairwise (list: 'T list) =
-            Microsoft.FSharp.Primitives.Basics.List.pairwise list
```

Comment:
OK - it's not a big deal but perhaps reverse these (just don't open Microsoft.FSharp.Primitives.Basics?) when you get the chance